Brick.ttl

```diff
@@ -3066,6 +3058,17 @@ brick:Max_Discharge_Air_Static_Pressure_Setpoint_Limit a owl:Class ;
         tag:Setpoint,
         tag:Static .
 
+brick:Max_Discharge_Air_Temperature_Setpoint a owl:Class ;
```

Comment:
What is this?

Reply:
These were technically already in the ontology under `setpoint.py`; all I did was adjust what they were a subclass of (which didn't make sense). They should be `Parameter` instead, so I can make that fix

Reply:
I think they should have been removed before, but not just moved to `Parameter`. Our choice was to use `_Limit` for such classes, if I recall correctly. But in this PR, I think your change makes sense. We can resolve this issue later if you prefer.

Reply:
Addressed in the most recent commit

Reply:
They weren't removed before, and their `Parameter`ized selves do exist in the `Parameter` hierarchy with the expected `_Limit` suffix